test/load/packer/consul-ami/README.md

```diff
@@ -0,0 +1,16 @@
+# Consul AMI
+
+## Quick start
+
+To build the Consul AMI:
+
+1. `git clone` this repo to your computer.
+2. Install [Packer](https://www.packer.io/).
+3. Configure your AWS credentials using one of the [options supported by the AWS
+   SDK](http://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html). Usually, the easiest option is to
+   set the `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `AWS_REGION` environment variables.
```

Comment:
The [region environment variable](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#environment-variables) is `AWS_DEFAULT_REGION`